js/scenes/AccountSettings/AccountSettingsContainer.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { ImagePickerIOS } from 'react-native';
+import AccountSettings from './AccountSettings';
+import { logout } from '../../config/helpers';
+import { 
+  updateEmailField, 
+  updatePasswordField,
+  updateFullnameField,
+  updateImageUrlField
+} from '../../redux/modules/user-forms';
+
+
+class AccountSettingsContainer extends Component {
+
+  static route = {
```

Comment:
if there's no `state` or life cycle methods, choose for a `functional component`

Reply:
Please ignore the AccountSettings components as it's still being worked on (included in the PR by accident during the squashing of commits).